props/prop_base.h

```diff
@@ -587,11 +587,15 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
     uint32_t now = millis();
     if (now - last_scan_id_ > BLADE_ID_SCAN_MILLIS) {
       last_scan_id_ = now;
+#if PROFFIEOS_LOG_LEVEL >= 500
+      size_t best_config = FindBestConfig(true);
```

Comment:
no need for if/else/endif just change `true` to `PROFFIEOS_LOG_LEVEL >= 500`

Reply:
So good. I have to start thinking this way more.